packages/formkit/src/password.ts

```diff
@@ -0,0 +1,13 @@
+import { type FormKitTypeDefinition } from '@formkit/core'
+import { token } from '@formkit/utils'
+import { text } from './text'
+
+/**
+ * Input definition for a password.
+ * @public
+ */
+export const password: FormKitTypeDefinition = {
+  ...text,
+  forceTypeProp: 'password',
```

Comment:
Why do we need all these `forceTypeProp`?

From my understanding - it takes preference before the `type` prop user would define, and it makes no practical sense to me.

<img width="575" alt="image" src="https://github.com/user-attachments/assets/6e4f3a8a-4b87-4ae4-99ce-ebd8f6ca58f7" />